packages/common/keyboard/src/util.ts

```diff
@@ -2,28 +2,68 @@
 // Copyright 2023 DXOS.org
 //
 
+import { getHostPlatform } from '@dxos/util';
+
 // https://www.w3.org/TR/DOM-Level-3-Events/#events-keyboardevents
 // https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key
 // https://support.apple.com/en-us/HT201236
 // https://support.apple.com/guide/mac-help/what-are-those-symbols-shown-in-menus-cpmh0011/mac
 
-// TODO(burdon): Show Mac/Windows/Linux variants.
-const symbols: Record<string, string> = {
-  // Mods.
-  ctrl: '⌃',
-  shift: '⇧',
-  alt: '⌥',
-  meta: '⌘',
+const alt: Record<string, string> = {
+  macos: '⌥',
+  ios: '⌥',
+  windows: 'Alt',
```

Comment:
the shortcut dialog current attempts to render these in a 24x24px container, which i guess wasn't going to work for everything anyway...